cedar-policy-core/src/parser/cst_to_ast.rs

```diff
@@ -169,7 +169,7 @@ impl Node<Option<cst::Policy>> {
         self.to_policy_template(id)
     }
 
-    /// Convert `cst::Policy` to an AST `StaticPolicy` or `Template`
+    /// Convert `cst::Policy` to an AST `InlinePolicy` or `Template`
```

Comment:
`InlinePolicy` does not exist. The correct terminology is `StaticPolicy`

Reply:
Oops, this got snuck in during rebasing. Will fix